src/scene/vcFlythrough.cpp

```diff
@@ -389,3 +396,17 @@ void vcFlythrough::UpdateCameraPosition(vcState *pProgramState)
     }
   }
 }
+
+void vcFlythrough::UpdateLinePoints()
+{
+  if (m_flightPoints.length > 1)
+  {
+    vcLineRenderer_DestroyLine(&m_pLine);
```

Comment:
What you had before was fine. Just check if its null.

Reply:
ok. it looks like createLine doesn't free it that's why I did it this way